infra/recipes/docker-compose/common/oauth/openhouse-realm.json

```diff
@@ -0,0 +1,85 @@
+{
+  "realm": "openhouse",
+  "enabled": true,
+  "users": [
```

Comment:
Is this common realm config to maintain predefined set of users and roles?

Reply:
Yes, sample realm config to maintain users and roles for local docker environment.